gum/guminterceptor.c

```diff
@@ -525,6 +533,30 @@ gum_interceptor_replace (GumInterceptor * self,
                          gpointer replacement_function,
                          gpointer replacement_data,
                          gpointer * original_function)
+{
+  return gum_interceptor_replace_ex (self, GUM_INTERCEPTOR_TYPE_DEFAULT,
+      function_address, replacement_function, replacement_data,
+      original_function);
+}
+
+GumReplaceReturn
+gum_interceptor_replace_fast (GumInterceptor * self,
+                              gpointer function_address,
+                              gpointer replacement_function,
+                              gpointer * original_function)
+{
+  return gum_interceptor_replace_ex (self, GUM_INTERCEPTOR_TYPE_FAST,
+      function_address, replacement_function, NULL,
+      original_function);
+}
+
+static GumReplaceReturn
+gum_interceptor_replace_ex (GumInterceptor * self,
```

Comment:
```suggestion
gum_interceptor_replace_full (GumInterceptor * self,
```

Err, or something like that... at least GLib uses `_full` in a few places. Another option is `gum_interceptor_replace_with_type`. What do you think?